Makefile

```diff
@@ -14,6 +14,10 @@ end-to-end-test:
 	sudo docker build -f ./test/end-to-end/Dockerfile -t weld/end-to-end:latest ./test/end-to-end/
 
 	make -C welder-deployment/ repos
+	git clone ../welder-web
```

Comment:
why not instead of all these commands `git clone ../welder-web welder-deployment/welder-web` ?